root/edit/details/AddRemoveAlias.js

```diff
@@ -0,0 +1,95 @@
+// @flow
```

Comment:
Copy the license header from here: https://github.com/metabrainz/musicbrainz-server/blob/de7d769042e189d4b64236da28146b4235aa903c/root/edit/details/AddArea.js#L1-L8 except replace "MetaBrainz Foundation" with your name. :)